song/app.js

```diff
@@ -94,6 +94,55 @@ app.post("/content", async (req, res, next) => {
   );
   res.status(201).json({ message: "postCreated" });
 });
+// 좋아요 기능
+app.post("/like", async (req, res, next) => {
+  const { userId, postId } = req.body;
+  await appDataSource.query(
+    `
+    INSERT INTO likes(
+      user_id,
+      post_id
+    ) VALUES (?, ?)
+    `,
+    [userId, postId]
+  );
+
+  res.status(201).json({ message: "likeCreated" });
+});
+
+app.patch("/posts", async (req, res) => {
+  const { content, userId } = req.body;
+  await appDataSource.query(
+    `UPDATE posts
+          SET 
+          content = ?
+          WHERE posts.user_id = ?
+          `,
+    [content, userId]
+  );
+  const update = await appDataSource.query(
+    `SELECT
+           users.id as userId,
+           users.name as userName,
+           posts.user_id as postingId,
+           posts.title as postingTitle,
+           posts.content as postingContent
+           FROM users LEFT JOIN posts ON users.id = posts.user_id     
+    `
+  );
+
+  res.status(201).json({ data: update });
+});
+
+app.delete("/posts/:postId", async (req, res) => {
+  const { postId } = req.params;
+  await appDataSource.query(
+    `DELETE FROM posts
+    WHERE posts.user_id = ?`,
```

Comment:
```suggestion
    WHERE posts.id = ?`,
```
변수를 기존의 userId와 달리 postId를 받도록 수정되었기 때문에 raw sql query도 수정되어야 합니다.